src/backend/config/routers.py

```diff
@@ -131,4 +135,11 @@ class RouterName(StrEnum):
             Depends(validate_authorization),
         ],
     },
+    RouterName.SCIM: {
+        "default": [
+            Depends(get_session),
+            Depends(ScimAuthValidation()),
+        ],
+        "auth": [],
```

Comment:
Auth dependencies only get add if user auth is setup, these endpoints should always be protected independently of that. Otherwise they would be open to the public if you do not have e.g.: OIDC configured.

Reply:
misunderstand how the dependency system work, also added to auth now